macros/hardware_functions/status_leds.cfg

```diff
@@ -180,12 +180,23 @@ gcode:
     {% endif %}
 
     {% if printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
-        _SET_LEDS_BY_NAME LEDS="logo" COLOR={status_color[color].logo} TRANSMIT={logo_transmit}
-        _SET_LEDS_BY_NAME LEDS="nozzle" COLOR={status_color[color].nozzle} TRANSMIT=1
+        {% if printer["gcode_macro _USER_VARIABLES"].status_leds_logo_led_name == "status_leds"%}
+            _SET_LEDS_BY_NAME LEDS="logo" COLOR={status_color[color].logo} TRANSMIT={logo_transmit}
+            _SET_LEDS_BY_NAME LEDS="nozzle" COLOR={status_color[color].nozzle} TRANSMIT=1
+        {% elif printer["gcode_macro _USER_VARIABLES"].status_leds_logo_led_name == "status_leds_effects"%}
+            {% set print_line = "Effect changed - logo: %s, nozzle: %s\n" % ("sb_logo_" + status_color[color].logo, "sb_nozzle_" + status_color[color].nozzle) %}
+            {action_respond_info(print_line)}
```

Comment:
is this needed? looks like some sort of info printing of the print?

Reply:
(i mean line 188)

Reply:
No I wouldn't have it on the finished implementation. It is there more for debug purposes and to make sure effects are triggered when they are supposed to. For example QGL effect wasn't working for me and printing out this info allowed me to find a reset to the ready state in conditional homing (_CG28 macro) outside its if condition that was messing things up. It was making the regular status led flicker and totally messing up the effects. This implementation is not completed yet. I just submitted to see if you guys were interested in me continuing pursuing this avenue, as I saw other people already attempting to integrate led effects into klippain.

Reply:
Yes, I'm all in for this PR 🚀 
Thanks a lot for your work!